src/window.rs

```diff
@@ -632,7 +632,8 @@ impl Window {
     ///
     /// ## Platform-specific
     ///
-    /// **Wayland:** - schedules a frame callback to throttle [`WindowEvent::RedrawRequested`].
+    /// - **Android / iOS / X11 / Web / Windows:** Unsupported.
```

Comment:
Should it say "Does nothing", "Not implemented" or "Unsupported" ?

Reply:
I think "Does nothing"

Reply:
`Unsupported` is fine, because we can add something in there later. 

you should also mention all the platforms we support.

Reply:
> you should also mention all the platforms we support.

`/// - **Wayland:** Schedules a frame callback to throttle [`WindowEvent::RedrawRequested`].`

This is not enough ?

Reply:
You miss `Orbital` and `macOS` in `Unsupported` list.